docs/mkdocs.yml

```diff
@@ -72,15 +77,10 @@ nav:
       - WatsonDiscovery: deep-dive/retrieval_models_clients/WatsonDiscovery.md
       - WeaviateRM: deep-dive/retrieval_models_clients/WeaviateRM.md
       - YouRM: deep-dive/retrieval_models_clients/YouRM.md
-    - Assertions: deep-dive/assertions.md
-    - Teleprompters:
-      - LabeledFewShot: deep-dive/optimizers/LabeledFewShot.md
-      - BootstrapFewShot: deep-dive/optimizers/bootstrap-fewshot.md
-      - BootstrapFinetune: deep-dive/optimizers/BootstrapFinetune.md
-      - Ensemble: deep-dive/optimizers/Ensemble.md
-      - BFRS: deep-dive/optimizers/bfrs.md
-      - CoPro: deep-dive/optimizers/copro.md
-      - MIProV2: deep-dive/optimizers/miprov2.md
+    - Data Handling:
```

Comment:
Can we push data handling to the top again please?

Reply:
done